### ISSUE TITLE
EUIV DLC Not Loading Through ModManager 

### ISSUE THREAD

**Issue by Trains6824** [bcssov/IronyModManager#287]
**Describe the bug**
Hello, I am trying to play EUIV through your mod manager, I have done so for all paradox game for over the last 2 years almost just fine and really do appreciate and love the mod manager you have made :)  

for some reason though, the EUIV game DLC is not loading when I run the game through your launcher, [not one of the DLC's is active. All not owned.]

it used to work just fine. I stopped playing EUIV for 4 months and when I came back and tried loading the game, it loads to the menu but says all the DLC are not owned even though I own them all.

I have disabled all my mods and ran it vanilla through the mod manager to remove the mods from the equation and I get the same result. I have also tried loading the game through the game launcher and the DLC shows up active and works just fine. 

**To Reproduce**
1.  not entirely sure as The Mod Manager seems to function fine beyond this issue.

**Expected behavior**
The mod Manager should load the game, with all the DLC + and Workshop mods I have added.

**Screenshots**

Below I have attached 3 pictures, 2 of irony mod manager vanilla settings with no mods and the menu of EUIV, and a pic of EUIV through the paradox launcher and how it should look.

Irony Mod Manager Game Menu:
![936 Aug  21, 2021 11 31 PM](https://user-images.githubusercontent.com/89328439/130341257-12de2b1d-e1f8-49a8-b4c4-e9b1d55114f6.jpg)
![938 Aug  21, 2021 11 34 PM](https://user-images.githubusercontent.com/89328439/130341259-e47c101c-516f-4848-9bce-b75074ea208e.jpg)

EUIV Paradox Laucher Game Menu:
![939 Aug  21, 2021 11 37 PM](https://user-images.githubusercontent.com/89328439/130341260-b53c20e5-3300-49d8-9d5b-92c8d038b14c.jpg)

**Logs**
If the application crashed add error logs related to the crash.
the game does not crash

**Version:**
- EUIV Version 1.31.5.2 Majapahit
 - Windows 10 Pro
 - Irony Version 1.17.1074

Regards, Trains6824
8/21/21

**Comment by bcssov** [comment]
Irony merely reads the DLC data and shows it to you and launch the game executable. What you're missing is probably online verification step that the game client performs (such as steam verification). If you're on Steam perhaps it is not started and there was no time to validate that you own these DLC, if you're using some other platform that other client is not launched to do the same thing.

**Comment by Trains6824** [comment]
> 
> 
> Irony merely reads the DLC data and shows it to you and launches the game executable. What you're missing is probably online verification step that the game client performs (such as steam verification). If you're on Steam perhaps it is not started and there was no time to validate that you own these DLCs, if you're using some other platform that other client is not launched to do the same thing.
> 
> TLDR; Game client performs an online validation check which seems to be missing in this case whether it be no steam running or something.

Hello, Bcssov, Thanks for the prompt reply! :) Yes, I own EUIV through stream lol. it appears apparently I hadn't played in so long that the launcher defaulted I think to an in-between state of mod lists I think because I applied a vanilla list, then applied a modded list, and reapplied a vanilla list and now it is loading all the DLC through the launcher again. not sure what exactly make everything click again but its fxied :)

**Comment by bcssov** [comment]
I'm lost, but glad you resolved it. Closing the ticket then.

**Comment by Trains6824** [comment]
> 
> 
> I'm lost, but glad you resolved it. Closing the ticket then.

so you think it was a steam registration issue then? not really sure then why it fixed it self so to speak as my steam was on the whole time and online.

**Comment by bcssov** [comment]
Steam validates DLC ownership, game does not do anything. This is as far as I'm aware how all of their games behave due to their games being available on multiple platforms (GOG, steam etc). So in your case it should be related to steam unless Paradox changed something in recent months.

**Comment by Trains6824** [comment]
i see, well Thanks for your communication and have a great rest of your day! :)
